ack/readers/awin_advertiser/cli.py

```diff
@@ -28,7 +28,10 @@
 @click.option("--awin-advertiser-id", required=True)
 @click.option("--awin-report-type", type=click.Choice(REPORT_TYPES), default=REPORT_TYPES[0])
 @click.option("--awin-region", required=True)
+@click.option("--awin-campaign", default=None)
 @click.option("--awin-timezone", required=True)
+@click.option("--awin-interval", default=None)
+@click.option("--awin-remove-tags", default=False)
```

Comment:
I think you should specify that the type here is boolean. It can avoid errors if people make mistakes. It's just a detail though

Reply:
this makes sense, yes, I have done this quickly